context/context.go

```diff
@@ -77,6 +77,11 @@ func (ctx *Ctx) LoadConfig(path *string) error {
 		log.Info("Fetching configurations tagged by: ", ctx.UserConfig.Tags)
 	}
 
+	// cache_path
+	if ctx.UserConfig.CachePath == "" {
+		log.Fatal("No cache directory was configured.")
```

Comment:
Shouldn't we have a default for this? Otherwise people who are upgrading to a new sidecar version but do not adjust the config file will have failing sidecars.